### PR TITLE
Conductivity Probe Code

### PR DESCRIPTION
**Pull request by aroraa68**
Added code for conductivity probe. Set a constant called EC_THRESH with some random value of 100 so it can be easily changed later

### CODE REVIEW

**Comment by ShyavanS** [comment]
Looks good for the most part! I'd say remove the calibration line at the end though, we won't be needing that apart from initial calibration with the sensor before comp so it doesn't need to be in the competition code. Also, remove the extrra temperature sensor code from last year. Now that we're using the conductivity probe to stop the car, we don't need the extra if statemenet and temperature difference calculations. We're just using the raw temperature to correct for any temperature-based drift in the conductivity readings.

File: Car_Code/cheme_car_datalogging_codebase/cheme_car_datalogging_codebase.ino [hunk 1]
```diff
@@ -376,6 +394,18 @@ void loop() // Loop (main loop)
...
+
+    // Indicate status to be finished
+    pixel.setPixelColor(0, 0, 0, 255);
+    pixel.show();
+
+    while (1)
+      ; // Do nothing for remainder of uptime
+  }
+  
   if (tempChange >= tempDiff)
```
Comment (ShyavanS) [inline Car_Code/cheme_car_datalogging_codebase/cheme_car_datalogging_codebase.ino]: Also won't need this and it's accompanying code to calculate tempDiff & tempChange. Any references to those should be removed.

File: Car_Code/cheme_car_datalogging_codebase/cheme_car_datalogging_codebase.ino [hunk 2]
```diff
@@ -388,4 +418,5 @@ void loop() // Loop (main loop)
     while (1)
       ; // Do nothing for remainder of uptime
   }
+  ec.calibration(voltage, temperatureC); // Calibration process
```
Comment (ShyavanS) [inline Car_Code/cheme_car_datalogging_codebase/cheme_car_datalogging_codebase.ino]: This won't be needed.

**Review by ShyavanS — CHANGES_REQUESTED** [review]
Looks good for the most part! I'd say remove the calibration line at the end though, we won't be needing that apart from initial calibration with the sensor before comp so it doesn't need to be in the competition code. Also, remove the extrra temperature sensor code from last year. Now that we're using the conductivity probe to stop the car, we don't need the extra if statement and temperature difference calculations. We're just using the raw temperature to correct for any temperature-based drift in the conductivity readings.

**Comment by aroraa68** [comment]
I removed the temperature change code and calibration code

**Review by ShyavanS — APPROVED** [review]
Looks good!

**Review by aithihya-k — APPROVED** [review]
looking good!

**Pull request merged**